### PR TITLE
typo

### PR DESCRIPTION
**Pull request by hatem20**
- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md)?
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS that we support](https://www.elastic.co/support/matrix#show_os)?
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

**Review by cbuescher — CHANGES_REQUESTED** [review]
Hi @hatem20, thanks for the PR. I'm not a native speaker but according to http://grammarist.com/usage/indexes-indices/ both "indexes" and "indices" are common plural forms of "index". I cannot find any evidence that the form you are using is correct though.

We seem to use both in our documentation, with an emphasis on "indices". If you would like to change your PR to that, I'd be okay merging it, but since we use both forms currently maybe we can also close this PR. Please push an update or close as you see fit.

File: docs/reference/getting-started.asciidoc [hunk 1]
```diff
@@ -526,7 +526,7 @@ PUT /customer/_doc/2?pretty
 // CONSOLE
 // TEST[continued]
 
-The above indexes a new document with an ID of 2.
+The above indeces a new document with an ID of 2.
```
Comment (DaveCTurner) [inline docs/reference/getting-started.asciidoc]: This is a verb, so `indexes` is the only correct spelling.

File: docs/reference/getting-started.asciidoc [hunk 2]
```diff
@@ -607,7 +607,7 @@ instead of deleting all documents with the Delete By Query API.
 
 In addition to being able to index, update, and delete individual documents, Elasticsearch also provides the ability to perform any of the above operations in batches using the {ref}/docs-bulk.html[`_bulk` API]. This functionality is important in that it provides a very efficient mechanism to do multiple operations as fast as possible with as few network roundtrips as possible.
 
-As a quick example, the following call indexes two documents (ID 1 - John Doe and ID 2 - Jane Doe) in one bulk operation:
+As a quick example, the following call indeces two documents (ID 1 - John Doe and ID 2 - Jane Doe) in one bulk operation:
```
Comment (DaveCTurner) [inline docs/reference/getting-started.asciidoc]: This is a verb, so `indexes` is the only correct spelling.

File: docs/reference/getting-started.asciidoc [hunk 3]
```diff
@@ -546,7 +546,7 @@ Note that in the above case, we are using the `POST` verb instead of PUT since w
 
 === Updating Documents
 
-In addition to being able to index and replace documents, we can also update documents. Note though that Elasticsearch does not actually do in-place updates under the hood. Whenever we do an update, Elasticsearch deletes the old document and then indexes a new document with the update applied to it in one shot.
+In addition to being able to index and replace documents, we can also update documents. Note though that Elasticsearch does not actually do in-place updates under the hood. Whenever we do an update, Elasticsearch deletes the old document and then indeces a new document with the update applied to it in one shot.
```
Comment (DaveCTurner) [inline docs/reference/getting-started.asciidoc]: This is a verb, so `indexes` is the only correct spelling.

**Review by DaveCTurner — CHANGES_REQUESTED** [review]
Both are fine as plurals of the noun "an index", and we do seem to prefer "indices" (e.g. the API includes `GET /cat/_indices`). "Indeces", despite seeming like a reasonable compromise, isn't ok.

However, as the third person singular of the verb "to index", only "indexes" is acceptable.

**Comment by hatem20** [comment]
I was just talking about consistency, but you are right both words are valid.

**Pull request closed without merging**